src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -22,14 +22,12 @@
 import com.google.api.codegen.SnippetSetRunner;
 import com.google.api.codegen.csharp.CSharpDiscoveryContext;
 import com.google.api.codegen.csharp.CSharpSnippetSetRunner;
-import com.google.api.codegen.discovery.config.java.JavaTypeNameGenerator;
-import com.google.api.codegen.discovery.transformer.java.JavaSampleMethodToViewTransformer;
+import com.google.api.codegen.discovery.config.nodejs.NodeJSTypeNameGenerator;
```

Comment:
Aren't  you doing this on top of the previous Java work? Shouldn't the analogous Java includes already be here?